### DISCUSSION TITLE
Resize all panes

### DISCUSSION THREAD

**Discussion by skerit** [wezterm/wezterm#2865] [Q&A]
When resizing the window, only 1 pane seems to grow & shrink. The other panes kind of stay the way they are.
For example, this setup:

![image](https://user-images.githubusercontent.com/755212/207846220-88972f0a-624b-4f3b-b182-45a72ce6d8c8.png)

Turns into this when resized:

![image](https://user-images.githubusercontent.com/755212/207846324-074dda94-d392-472e-92cd-85d2fd24fc7d.png)

How can I configure it so each pane will stick to 33% width & 50% height?

**Comment by lalitkapoor** [comment]
@skerit did you ever figure this out?

**Reply by skerit** [reply]
Unfortunately not.

**Reply by lalitkapoor** [reply]
Thanks for getting back to me @skerit! Did you find an acceptable work around? tmux? go with another terminal emulator?

**Reply by skerit** [reply]
I'm still just using Wezterm. It's an annoying issue, but I don't resize my terminal frequently enough for it to be a showstopper.
